build.json

```diff
@@ -0,0 +1,4 @@
+{
+    "minimumEJSVersion": "4.0.13",
+    "version": "2.0.0"
```

Comment:
Is there a reason we're starting on 2? Not complaining or asking for a change (just yet), just wondering.

Reply:
I was working with the idea that the current way things are done would be version 1.0.0. - I'm not married to this assumption though, so happy to reset the number back to 1.0.0 as it's not used anywhere yet.